pandas/core/generic.py

```diff
@@ -1016,6 +1016,64 @@ def __setstate__(self, state):
     # ----------------------------------------------------------------------
     # I/O Methods
 
+    _shared_docs['to_excel'] = """
+    Write %(klass)s to a excel sheet
+
+    .. versionadded:: 0.20.0
```

Comment:
@jorisvandenbossche how will you name this key in `_shared_doc_kwargs`?
What value will you put? `.. versionadded:: 0.20.0` or `0.20.0`?